### ISSUE TITLE
Can't focus out when cursor is at the end of paragraph

### ISSUE THREAD

**Issue by z-ax** [yabwe/medium-editor#400]
Sounds a bit weird, but you can check it as well here:
http://jsfiddle.net/efqoqnqe/2/

When cursor is at the end of the first paragraph, you can't focus inside Header editor or any other place.
But, as I checked, `blur` event is triggered fine.

**Comment by daviferreira** [comment]
Very weird :confused:

**Comment by nmielnik** [comment]
This may have been fixed by #510, @z-ax can you verify?

**Comment by nmielnik** [comment]
Just double checked myself, looks like this is still happening...

I noticed this was happening inside a bootstrap modal, issue #315 tracks some issues that are happening specifically inside a bootstrap modal, these very well could be related

**Comment by nmielnik** [comment]
@z-ax would you be able to see if the fix outlined in #315 resolves this issue?

**Comment by z-ax** [comment]
@nmielnik I updated JSFiddle to the latest BS version, but it anyways didn't show the Modal even :) So, added a class update `.modal.fade.in`, checked and seems like issue is gone:
http://jsfiddle.net/efqoqnqe/6/

Checked both variants, including creation of Medium instance on `shown.bs.modal` event - all works as expected now, seems fixed.

Thanks for your work!

P.S. Don't work on a project with editor for a while, so sorry for a late reply, just realised I have this open issue.
